src/Promenade.Web/Views/Home/LocationDetails.cshtml

```diff
@@ -0,0 +1,214 @@
+                    <td class="hours">
+                        <strong>Mon. - Sun.</strong> Closed<br />
+                    </td>
+                </tr>
+            </tbody>
+        </table>
+        <h2>Information</h2>
+        <p>@Model.Location.Description</p>
+        <div>
+            <div>
+                <a href="@Model.Location.EventLinkId" target="_blank">Browse Events</a>
+            </div>
+            <div>
+                <a href="@Model.Location.SubscriptionLinkId" target="_blank">Send Me Events</a>
+            </div>
+        </div>
+        <div class="row branch-features text-center">
+            @foreach (var feature in Model.LocationFeatures)
+            {
+                <text>
+                    <div class="col-xs-2">
+                        @if (feature.RedirectUrl != null)
+                        {
+                            <a href="@feature.RedirectUrl" target="_blank">
+                                <span class="fa-stack fa-2x" style="color:#007bff">
+                                    <span class="fa fa-square fa-stack-2x"></span>
+                                    <span class="@feature.FontAwesome">
+                                        @if (feature.Name == "7-Day Express")
+                                        {
+                                            <text><strong style="color:white;">7</strong></text>
+            }
+                                    </span>
+                                </span>
+                            </a>
+                            <div>
+                                <a href="@feature.RedirectUrl">@feature.Name</a>
+                            </div>
+                        }
+                        else
+                        {
+                            <a href="@feature.RedirectUrl" target="_blank">
+                                <span class="fa-stack fa-2x" style="color:#007bff">
+                                    <span class="fa fa-square fa-stack-2x"></span>
+                                    <span class="@feature.FontAwesome">
+                                        @if (feature.Name == "7-Day Express")
+                                        {
+                                            <text><strong style="color:white;">7</strong></text>
+                                        }
+                                    </span>
+                                </span>
+                            </a>
+                            <div>
+                                <a class="bring-modal" href="/Locations/@Model.Location.Stub/@feature.Stub" data-toggle="modal" data-target="#myModal" data-feature="@feature.Stub">@feature.Name</a>
+                            </div>
+                        }
+
+                    </div>
+                    </text>
+                }
+        </div>
+        @if (Model.NearByLocations.Count() > 0){
+        <div>
+            <a name="nearby"></a>
+            <h2>Nearby Libraries</h2>
```

Comment:
Don't use the word "Libraries" here - if we need it on the page it has to come from the database.